### PR TITLE
Minor English improvements

### PR DESCRIPTION
**Pull request by Toflar**
I'm not an English native speaker either but I tried to improve some stuff on the docs. Feel free to take whatever you like or ditch it completely. That is just my way to give back for the awesome project!
Now I only need the protocol feature for Windows :-P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,33 +22,34 @@ Real project example — [onshape-desktop-shell](https://github.com/develar/onsh
...
+## Quick setup guide
 
-2. Specify [build](https://github.com/electron-userland/electron-builder/wiki/Options#build) field in the development `package.json`:
+1. Specify the standard fields in the application `package.json` — [name](https://github.com/electron-userland/electron-builder/wiki/Options#AppMetadata-name), `description`, `version` and [author](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) (for Linux [homepage](https://github.com/electron-userland/electron-builder/wiki/Options#AppMetadata-homepage) and [license](https://github.com/electron-userland/electron-builder/wiki/Options#AppMetadata-license) are also required).
+
+2. Specify the [build](https://github.com/electron-userland/electron-builder/wiki/Options#build) key in the development `package.json`:
```
Comment (develar) [inline README.md]: Why `key` is used here if before `field` term was used?

File: README.md [hunk 2]
```diff
@@ -58,49 +59,49 @@ For a production app you need to sign your application, see [Where to buy code s
...
 
-3. Create directory `build` in the root of the project and put your `background.png` (MacOS DMG background), `icon.icns` (MacOS app icon) and `icon.ico` (Windows app icon).
+3. Create a directory `build` in the root of the project and save a `background.png` (MacOS DMG background), `icon.icns` (MacOS app icon) and `icon.ico` (Windows app icon) into it.
 
-   <a id="user-content-linuxIcon" class="anchor" href="#linuxIcon" aria-hidden="true"></a>Linux icon set will be generated automatically on the fly from the MacOS `icns` file (or you can put them into the `build/icons` directory — filename must contains size (e.g. `32x32.png`)).
+   <a id="user-content-linuxIcon" class="anchor" href="#linuxIcon" aria-hidden="true"></a>The Linux icon set will be generated automatically based on the MacOS `icns` file (or you can put them into the `build/icons` directory if you want to specify them yourself. The filename must contain the size (e.g. `32x32.png`) of the icons).
```
Comment (develar) [inline README.md]: I guess not `of the icons` (plural), but `of the icon` (singular).

File: README.md [hunk 3]
```diff
@@ -58,49 +59,49 @@ For a production app you need to sign your application, see [Where to buy code s
...
-4. Add [scripts](https://docs.npmjs.com/cli/run-script) to the development `package.json`:
+4. Add the [scripts](https://docs.npmjs.com/cli/run-script) key to the development `package.json`:
     ```json
     "scripts": {
       "pack": "build --dir",
       "dist": "build"
     }
     ```
-    And then you can run `npm run dist` (to package in a distributable format (e.g. dmg, windows installer, deb package)) or `npm run pack` (useful to test).
+    Then you can run `npm run dist` (to package in a distributable format (e.g. dmg, windows installer, deb package)) or `npm run pack` (only generates the package directory without really packaging it. Very useful for testing purposes).
```
Comment (develar) [inline README.md]: Don't like `Very useful`, may be just `Useful`?

File: README.md (hunk 1 above)
Comment (Toflar) [inline README.md]: I just felt like `key` is more appropriate than `field`. It's JSON and for JSON  you wouldn't say "field -> value" but "key -> value" instead. Feel free to use `field` if you want to. I just changed everything that sounded strange to me :D

File: README.md (hunk 2 above)
Comment (Toflar) [inline README.md]: True :) Fixing... :)

**Comment by develar** [comment]
> Now I only need the protocol feature for Windows

Ok, In 2 days ;)

**Comment by Toflar** [comment]
> Ok, In 2 days ;)

Haha, this was not a request, just a broad hint :-D Thank you for you awesome work!

File: README.md (hunk 1 above)
Comment (Toflar) [inline README.md]: Want me to change it back to field?

File: README.md [hunk 4]
```diff
@@ -58,49 +59,49 @@ For a production app you need to sign your application, see [Where to buy code s
...
+Consider using [Nuts](https://github.com/GitbookIO/nuts) (uses GitHub as a back end to store the assets), [Electron Release Server](https://github.com/ArekSredzki/electron-release-server) or [Squirrel Updates Server](https://github.com/Aluxian/squirrel-updates-server).
+See the [Publishing Artifacts](https://github.com/electron-userland/electron-builder/wiki/Publishing-Artifacts) section of the [Wiki](https://github.com/electron-userland/electron-builder/wiki) for more information on how to configure your CI environment for automated deployments.
 
-For windows consider only [distributing 64-bit versions](https://github.com/electron-userland/electron-builder/issues/359#issuecomment-214851130).
+For Windows consider only [distributing 64-bit versions](https://github.com/electron-userland/electron-builder/issues/359#issuecomment-214851130).
 
 # CLI Usage
-Execute `node_modules/.bin/build --help` to get actual CLI usage guide.
+Execute `node_modules/.bin/build --help` to get a detailed CLI usage guide.
```
Comment (develar) [inline README.md]: No, it is `actual`, not `detailed`. i.e. here maybe outdated info, but to get actual (effective), you need to execute `--help`.

File: README.md [hunk 5]
```diff
@@ -139,7 +140,7 @@ Examples:
 ```
 
 # Programmatic Usage
-See `node_modules/electron-builder/out/electron-builder.d.ts`. [Typings](https://github.com/Microsoft/TypeScript/wiki/Typings-for-npm-packages) is supported.
+See `node_modules/electron-builder/out/electron-builder.d.ts`. [Typings](https://github.com/Microsoft/TypeScript/wiki/Typings-for-npm-packages) are supported.
```
Comment (develar) [inline README.md]: `is` is correct :) singular — https://github.com/typings/typings — `Typings is the simple way to manage and install TypeScript definitions.`.

Comment (Toflar) [inline README.md]: Oh, ok :) Fixing :)

**Comment by Toflar** [comment]
Meh, seems like I need to checkout the repository anyway now. I think I'll squash the commits, otherwise the repo's commit history's going to be cluttered :D

File: README.md (hunk 1 above)
Comment (develar) [inline README.md]: Maybe just remove this word at all? `Specify build in the...`?

Comment (Toflar) [inline README.md]: Like https://github.com/electron-userland/electron-builder/pull/704/commits/8e9f7750d35dfe5f762921b7cc528331c5f252a2? :)

Comment (develar) [inline README.md]: Yep!

**Comment by develar** [comment]
> I think I'll squash the commits, otherwise the repo's commit history's going to be cluttered

It will be performed automatically on merge. Don't worry about it.

**Comment by Toflar** [comment]
Ah true, forgot about that newly introduced feature.

**Pull request merged**